man/build_manpage.py

```diff
@@ -142,9 +142,8 @@ def run(self):
         manpage.append(self._write_header())
         manpage.append(self._write_options())
         manpage.append(self._write_footer())
-        stream = open(self.output, "w")
-        stream.write("".join(manpage))
-        stream.close()
+        with open(self.output, mode="w") as stream:
+            stream.write("".join(manpage))
```

Comment:
This build script is just copied into our repository from https://github.com/andialbrecht/build_manpage  In my pylint cleanup patch, I excluded this file instead of fixing it in our copy so that we didn't get out of sync.  OTOH, the upstream looks dead so maybe changing it is the right thing to do...

(No changes needed, just something to think about).

Reply:
I'd not be worried about diverging since as you mentioned it's not maintained anymore.